src/System.Diagnostics.Debug/tests/DebugTests.cs

```diff
@@ -9,6 +9,7 @@ namespace System.Diagnostics.Tests
     public class DebugTests
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Net46)]
         public void Asserts()
         {
             VerifyLogged(() => { Debug.Assert(true); }, "");
```

Comment:
> Rant: This is why I don't like InternalsVisibleTo :(

This is why I don't like that we removed members like Debug.Listeners; this hack wouldn't have been necessary with that.